### ISSUE TITLE
post punctuator steps? 

### ISSUE THREAD

**Issue by cozec** [ottokart/punctuator2#12]
Question about steps after this:
    cat data.dev.txt | python punctuator.py <model_path> <model_output_path>

We get a text file have the result with ',COMMA' and '.PERIOD' etc inside. To generate final result, we assume following steps:
1. replace punc with real punc
2. Capitalize the previous word after .PERIOD'

Is this the right understanding?

**Comment by ottokart** [comment]
Yes, that's about correct (?QUESTIONMARK and !EXCLAMATIONMARK should also be taken into account).

I added a conversion script with the last commit. You can use it like this:
```python convert_to_readable.py <model_output_path> <readable_output_path> <1/0 - add newlines at end-of-sentence>```

**Comment by cozec** [comment]
Thanks a lot! Just added one more 'period' at the last line.

**Comment by MeteorBurn** [comment]
"convert_to_readable.py" deletes data from source file and creates a new one with repleced content
How to change "convert_to_readable.py" to replace content in the same file?

I'm newbie)

**Comment by acerock6** [comment]
@cozec can you help me train a model with a processed file that I have?
Thanks.
